Easydict/App/InfoPlist.xcstrings

```diff
@@ -0,0 +1,60 @@
+{
+  "sourceLanguage" : "en",
```

Comment:
can we use `Localizable.xcstrings`, why add a new localize strings file

Reply:
We cando that for the About view since we're just extracting string from the catalog, but the app info in /Applications won't be able to show the localized result based on the system language (like what we have now)
<img width="746" alt="截屏2024-06-04 22 45 12" src="https://github.com/tisfeng/Easydict/assets/89069957/442b6d7e-da2a-4065-abf8-2d2614ceb1be">

In my understanding, Since apps still uses `Localizable.strings` under the hood when is compiled, it can only extract strings for Info.plist from `InfoPlist.strings` files. Here's the documentation on how to localize `plist` files

https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/AboutInformationPropertyListFiles.html